candi/openapi/cluster_configuration.yaml

```diff
@@ -12,32 +22,41 @@ apiVersions:
       podSubnetCIDR: 10.244.0.0/16
       serviceSubnetCIDR: 192.168.0.0/16
       kubernetesVersion: "1.21"
-      clusterDomain: test
+      clusterDomain: k8s.internal
       clusterType: "Cloud"
       cloud:
-        prefix: test
+        prefix: k8s-dev
         provider: Yandex
       packagesProxy:
-        uri: https://example.com
-        username: user
-        password: passwd
+        uri: https://packages-proxy.company.my
+        username: "proxy-username"
+        password: "proxy-p@ssw0rd"
     properties:
       apiVersion:
         type: string
+        description: Version of the Deckhouse API.
         enum: [deckhouse.io/v1, deckhouse.io/v1alpha1]
       kind:
         type: string
         enum: [ClusterConfiguration]
       clusterType:
         type: string
+        description: |
+          Type of the cluster infrastructure:
+          - `Static` — a bare metal cluster (physical servers);
+          - `Cloud` — a cluster, deployed on the resources of a cloud provider.
         enum: [Cloud, Static]
       cloud:
         type: object
+        description: |
+          Cloud provider-related settings (if the `Cloud` [clusterType](#parameters-clustertype) is used).
         required: [provider]
         additionalProperties: false
         properties:
           provider:
             type: string
+            description: |
+              Type of the cloud provider.
```

Comment:
hmm. Maybe simplify it and use only `Cloud provider`?

Reply:
Paraphrased